repotracker/github_poller.go

```diff
@@ -218,6 +220,16 @@ func (gRepoPoller *GithubRepositoryPoller) GetRevisionsSince(revision string, ma
 		return []model.Revision{}, revisionError
 	}
 
+	if len(revisions) == 0 {
+		grip.Info(message.Fields{
+			"source":        "github poller",
+			"message":       "no new revisions",
+			"last_revision": revision,
+			"project":       gRepoPoller.ProjectRef,
+			"commits":       commits,
```

Comment:
This data structure contains https://godoc.org/github.com/google/go-github/github#Commit, which is recursive. (Its Parents field is a slice of `Commit` objects.) Is this potentially too large of an object to log to Splunk?

Reply:
That's a good point.
With regard to the recursive nature, it looks like GitHub only fills in the SHA, [patch_]url, and html_url of the parents.
Still, it is still pretty large. I reduced to it just include the SHAs.